### DISCUSSION TITLE
Iota stacking not generate asmb

### DISCUSSION THREAD

**Discussion by JAVIVILIS** [iotaledger/firefly#6866] [Help & Support]
Hi all, since yesterday I started with iota stacking to generate asmb, transaction seems fine but dont reflect asmb increase... is it working or not? The asmb tokens are frozen in previous amount stacking... thanks for you comments

**Comment by Raivr-dev** [comment]
Same here. Using version 1.7.2 on Windows, I can see the staking transaction in the Transactions list and the Staking page says all funds are staked, but no rewards coming in (yet).

**Comment by MiKeF05** [comment]
Make sure to have a stable internet connection. Otherwise you don´t see any rewards. Make also sure to be connected to a node that tracks the staking period.

**Comment by JAVIVILIS** [comment]
Solved with New app version, once upgraded, the stacking is working properly. Thanks and regards.
